packages/ffmpeg.cmake

```diff
@@ -110,8 +110,10 @@ ExternalProject_Add(ffmpeg
         --disable-vaapi
         --disable-vdpau
         --disable-videotoolbox
+        --disable-ptx-compression
         --disable-decoder=libaom_av1
         --extra-cflags='-Wno-error=int-conversion'
+        --nvccflags='-O3'
```

Comment:
I've read that O3 is actually the default optimization level for nvcc...are you sure this is doing anything?